src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 200) {
+    this.health = health;
+    this.name = 'Animal';
```

Comment:
The name property of the `Animal` class should be initialized with the name passed to the constructor, not hardcoded as 'Animal'.